andela/ah-frontend-thanos
# #161348762 Users can like or dislike an Article

#### What does this PR do?
Adds feature for liking and disliking an Article

#### Description of Task to be completed?
- Display a specific article (based on the id from the URL)
- Add Like and Dislike icons for each article
- Collects the like and dislike counts for each article
- Updates like/dislike in case a user changes their mind about an article

#### How should this be manually tested?
- Go to the article's URL: `/articles/:articleId` e.g. `/articles/12`
- You should be able to see that articles full details
- You should be able to like / dislike that article

#### Any background context you want to provide?
You do not need to be logged in to view an article but You need to be logged in to like or dislike an article

#### What are the relevant pivotal tracker stories?
[#161348762](https://www.pivotaltracker.com/story/show/161348762)

#### Screenshots (if appropriate)
*Full article details page*
![image](https://user-images.githubusercontent.com/36793346/49583492-6ff4da00-f969-11e8-8400-8ae3babdd752.png)

*Like, Dislike options on the article's page*
![image](https://user-images.githubusercontent.com/36793346/49583522-8d29a880-f969-11e8-8d56-c0b98fbdb91d.png)
